src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -270,23 +270,15 @@
   </div>
 </template>
 
-<script lang="ts" setup>
+<script setup lang="ts">
 import { computed, nextTick, onBeforeUnmount, onMounted, ref, watch, PropType } from 'vue'
 
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
 import { copyTextToClipboard } from '@/utilities/copyTextToClipboard'
 import { debounce } from '@/utilities/debounce'
 import { Command, ShortcutManager } from '@/utilities/ShortcutManager'
-
-export type CodeBlockEventData = {
-  preElement: HTMLElement
-  codeElement: HTMLElement
-  code: string
-  language: string
-  query: string
-  matchingLineNumbers: number[]
-}
+import { CodeBlockEventData, CommandKeywords, Theme } from '@/types/codeBlock'
```

Comment:
Always import from the root `types`
```suggestion
import type { CodeBlockEventData, CommandKeywords, Theme } from '@/types'
```